### PR TITLE
JNG-4472 release judo jsl springboot archetype

### PR DESCRIPTION
**Pull request by gaborflorian**
<table>
<td>
  <a href="https://blackbelt.atlassian.net/browse/JNG-4472" title="JNG-4472" target="_blank">Sub-taskJNG-4472</a>  Release judo-jsl-springboot-archetype
  </td></table>
  <br />

### CODE REVIEW

File: src/main/resources/META-INF/maven/archetype-metadata.xml [hunk 1]
```diff
@@ -29,10 +29,10 @@
         </requiredProperty>
 
         <!-- DO NOT REFORMAT BECAUSE REPLACER PLUGIN WILL FAIL-->
-        <requiredProperty key="jslSpringbootVersion"><defaultValue>JSL-SPRINGBOOT-PLACEHOLDER</defaultValue></requiredProperty>
+        <requiredProperty key="jslSpringbootVersion"><defaultValue>1.0.4.20230221_041641_66ea6c71_develop</defaultValue></requiredProperty>
 
         <!-- DO NOT REFORMAT BECAUSE REPLACER PLUGIN WILL FAIL-->
-        <requiredProperty key="jslTatamiVersion"><defaultValue>JSL-TATAMI-PLACEHOLDER</defaultValue></requiredProperty>
+        <requiredProperty key="jslTatamiVersion"><defaultValue>1.1.4.20230221_041625_ba9cc1a5_develop</defaultValue></requiredProperty>
```
Comment (bencegelei) [inline src/main/resources/META-INF/maven/archetype-metadata.xml]: This looks a bit suspicious part 1

File: src/test/resources/projects/basic/archetype.properties [hunk 2]
```diff
@@ -3,8 +3,8 @@ applicationName=ShopApplication
 groupId=hu.blackbelt
 artifactId=shop
 package=hu.blackbelt.shop
-jslSpringbootVersion=JSL-SPRINGBOOT-VERSION-PLACEHOLDER
-jslTatamiVersion=JSL-TATAMI-VERSION-PLACEHOLDER
+jslSpringbootVersion=1.0.4.20230221_041641_66ea6c71_develop
+jslTatamiVersion=1.1.4.20230221_041625_ba9cc1a5_develop
```
Comment (bencegelei) [inline src/test/resources/projects/basic/archetype.properties]: This looks a bit suspicious part 2

**Review by bencegelei — CHANGES_REQUESTED** [review]

File: src/main/resources/META-INF/maven/archetype-metadata.xml (hunk 1 above)
Comment (robertcsakany) [reply src/main/resources/META-INF/maven/archetype-metadata.xml]: It is replkaced by compilayion. Do not commit these changes.

File: src/test/resources/projects/basic/archetype.properties (hunk 2 above)
Comment (robertcsakany) [reply src/test/resources/projects/basic/archetype.properties]: It is replkaced by compilayion. Do not commit these changes.

**Review by robertcsakany — CHANGES_REQUESTED** [review]

File: src/test/resources/projects/basic/archetype.properties (hunk 2 above)
Comment (gaborflorian) [reply src/test/resources/projects/basic/archetype.properties]: [9b0620e](https://github.com/BlackBeltTechnology/judo-jsl-springboot-archetype/pull/31/commits/9b0620e949946d45b3567e895c56a555455b4774)

File: src/main/resources/META-INF/maven/archetype-metadata.xml (hunk 1 above)
Comment (gaborflorian) [reply src/main/resources/META-INF/maven/archetype-metadata.xml]: [9b0620e](https://github.com/BlackBeltTechnology/judo-jsl-springboot-archetype/pull/31/commits/9b0620e949946d45b3567e895c56a555455b4774)

**Review by robertcsakany — APPROVED** [review]

**Review by bencegelei — APPROVED** [review]

**Review by fmagnucz — APPROVED** [review]

**Pull request merged**